pandera/schemas.py

```diff
@@ -287,6 +289,35 @@ def __str__(self):
     def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
+    def add_columns(self,
+                    extra_schema_cols: Dict[str, Any]) -> 'DataFrameSchema':
```

Comment:
I wonder if we should be more strict here with a type signature of `Dict[str, Column]`? (unless we think at some point a subclass/other class can be used in the schema definition)

Reply:
Done - had to use forward declaration as discussed here: https://stackoverflow.com/questions/50735538/interdependence-of-2-classes-with-type-hint-in-python